components/constants.js

```diff
@@ -0,0 +1,5 @@
+const CONSTANTS = {
```

Comment:
This file can be removed now

Reply:
I removed it

Reply:
@eang0521 bump to remove this file. If it still shows up in the UI, it means that it wasn't removed from the PR/commits.

You can do so by doing `git rm` the file so that it gets staged.

I have a set of aliases in my `~/.gitconfig` file:

```
[alias]
    br = branch
    ci = commit
    co = checkout
    st = status
    # list files
    ls = ls-files
    # list deleted files
    lsd = ls-files --deleted
    # remove deleted files
    rmd = !git rm `git ls-files --deleted`
    master = checkout master
    gh-pages = checkout gh-pages
    dsf = "!f() { [ \"$GIT_PREFIX\" != \"\" ] && cd "$GIT_PREFIX"; git diff --color $@ | diff-so-fancy | less --tabs=4 -RFX; }; f"
```

`git rmd` does the trick for me. Over time, you'll want to customize your `~/.gitconfig` file to your personal liking.